editor/plugins/tiles/tile_set_atlas_source_editor.cpp

```diff
@@ -2578,17 +2563,17 @@ TileSetAtlasSourceEditor::TileSetAtlasSourceEditor() {
 	tile_atlas_view->add_control_over_alternative_tiles(alternative_tiles_control_unscaled, false);
 	alternative_tiles_control_unscaled->set_mouse_filter(Control::MOUSE_FILTER_IGNORE);
 
-	tile_atlas_view_missing_source_label = memnew(Label);
```

Comment:
If I understand correctly, this was unused right ? If so removing it is fine.

Reply:
Yes, it's never used.